PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1520,6 +1520,56 @@ def miniAODConfigurationPre(self, process, patMetModuleSequence, postfix):
                              )
             getattr(process,"patCaloMet").addGenMET = False
 
+            ##adding the necessary chs and track met configuration
+            task = getPatAlgosToolsTask(process)
+
+            pfChs = cms.EDFilter("CandPtrSelector", src = cms.InputTag("packedPFCandidates"), cut = cms.string("fromPV(0)>0"))
```

Comment:
please define a helper function (eg, a cfi) to avoid the code duplication

Reply:
this particular code is due for a overall cleaning / splitting and I would prefer to address it in a non time-critical period. when doing this overall code update, this comment can be addressed too.

Reply:
when is that planned?

Reply:
soon as we have an actual reco contact in the group. truthfully, this call has been open for roughly 5 months now with absolutely no interest from the collaboration. however I fully acknowledge the importance of simplifying and partitioning this code, therefore it will be highest priority.